README.md

```diff
@@ -124,10 +124,11 @@ Kor provides various subcommands to identify and list unused resources. The avai
 ```
       --delete                       Delete unused resources
   -l, --exclude-labels strings       Selector to filter out, Example: --exclude-labels key1=value1,key2=value2. If --include-labels is set, --exclude-labels will be ignored.
-  -e, --exclude-namespaces strings   Namespaces to be excluded, split by commas. Example: --exclude-namespaces ns1,ns2,ns3. If --include-namespaces is set, --exclude-namespaces will be ignored.
+      --exclude-namespaces strings   Namespaces to be excluded, split by commas. Example: --exclude-namespace ns1,ns2,ns3. If --include-namespace is set, --exclude-namespaces will be ignored.
```

Comment:
In commit `da2b1fe` we've updated README with plural form of flags (`--exclude/include-namespaces`).
Please adjust here as well.

Reply:
Done 👍